--- a/node_modules/zotero-plugin-toolkit/dist/basic.js
+++ b/node_modules/zotero-plugin-toolkit/dist/basic.js
@@ -52,11 +52,12 @@
             _mainWindow: undefined,
             _plugin: undefined,
         };
-        if (typeof globalThis.ChromeUtils?.import !== "undefined") {
-            const { ConsoleAPI } = ChromeUtils.import("resource://gre/modules/Console.jsm");
+        try {
+            const { ConsoleAPI } = ChromeUtils.importESModule("resource://gre/modules/Console.sys.mjs");
             this._console = new ConsoleAPI({
                 consoleID: `${this._basicOptions.api.pluginID}-${Date.now()}`,
             });
+        } catch (e) {
+            // Console not available
         }
         this.updateOptions(data);
     }
--- a/node_modules/zotero-plugin-toolkit/dist/utils/pluginBridge.js
+++ b/node_modules/zotero-plugin-toolkit/dist/utils/pluginBridge.js
@@ -26,7 +26,7 @@
         }
     }
     initializePluginBridge() {
-        const { AddonManager } = ChromeUtils.import("resource://gre/modules/AddonManager.jsm");
+        const { AddonManager } = ChromeUtils.importESModule("resource://gre/modules/AddonManager.sys.mjs");
         const Zotero = BasicTool.getZotero();
         const pluginBridgeExtension = {
             noContent: true,
